tests/jsUnit/global/navbar.spec.js

```diff
@@ -0,0 +1,51 @@
+import navbar from '../../../resources/js/globals/navbar'
+import Vuex from 'vuex'
+import User from '../../../resources/js/store/modules/user/index'
+import { shallowMount, createLocalVue } from '@vue/test-utils'
+
+const localVue = createLocalVue()
+localVue.use(Vuex)
+
+describe('navbar.vue', () => {
+  let store
+  let state
+
+  state = {
+    user: {
+      token: '123rke4',
+      username: 'csunStudent123'
+    },
+    correctLoginInfo: true
+  }
+
+  beforeEach(() => {
+    store = new Vuex.Store({
+      modules: {
+        User: {
+          state: User.state,
+          actions: User.actions,
+          mutations: User.mutations
+        }
+      }
+    })
+  })
+  describe('user/actions/clearUserData', () => {
```

Comment:
Move back action and mutation tests to the vuex file, but mock their return value in this component as it still needs a vuex instance

Reply:
yeee yee